nss-day-cohort-41/tabloidcli-dr-teeth-the-electric-mayhem
# Console COLOR BRANCH PR

# Console COLOR BRANCH PR Pull Request (x1 TICKETS):


->Console COLOR BRANCH PR
..TICKET: Application BackGround


No Dependencies, the changes to the blog section should not affect any other section….as tested with Authors section. Currently all tests come back clear and without error in log.

Fixes # (issue)
## Type of change
-  Added Files: None....added to MainMenu..
- New feature: The console now has a new color arrangement.....both in its background and foreground.

# Testing Instructions
Upon running: dotnet run…
-Press any key.. (Menu Header will appear)
-Press any Key.. (Color ChoiceMenu will display)
-Press 1-4 or 0 (To select your color scheme)
- Enjoi
-Main Menu will follow


# Checklist: WENT OVER WITH THE TEAM, DEMO'd
- [ yes ] My code follows the style guidelines of this project
- [ yes ] I have performed a self-review of my own code
- [ yes ] I have commented my code, particularly in hard-to-understand areas
- [ yes ] My changes generate no new warnings or errors
- [ yes ] I have added test instructions that prove my fix is effective or that my feature works
